File: Course_01/C Language/EstruturaRepetitiva/media_idades.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int idade, soma, count;
    double media;

    printf("Digite as idades:\n");
    scanf("%d", &idade);

    soma = 0;
    count = 0;

    if (idade > 0) {
        while (idade > 0) {
        count++;
        soma = soma + idade;
        scanf("%d", &idade);
    }

        media = (double) soma / count;

        printf("MEDIA = %.2lf", media);
    }
    else {
        printf("IMPOSSIVEL CALCULAR");
    }
    return 0;
}

File: Course_01/Python/EstruturasRepetitivas/media_idades.py
print("Digite as idades:")
idade = int(input())
if idade < 0:
    print("Impossível calcular")
else:
    soma = 0
    contagem = 0

    while idade > 0:
        soma += idade
        contagem += 1
        idade = int(input())

    media = soma / contagem

    print(f"Média = {media:.2f}")
